roles/post/tasks/motd.yml

```
@@ -0,0 +1,17 @@
+---
+# motd and creds gen
+- name: copy MOTD template to /etc/motd
+  template:
+    src: templates/motd.j2
+    dest: /etc/motd
+
+- name: writing jitsi creds into file
+  copy:
+    dest: "{{ item.user }}/.credentials"
+    mode: '0600'
+    content: |
+      sudo username: {{ username }}
+      sudo password: {{ password }}
+  loop:
+    - {user: "/root"}
+    - {user: "/home/{{ username }}"}
```

```suggestion
- name: writing jitsi creds into file
  copy:
    dest: "/home/{{ username }}/.credentials"
    mode: '0600'
    owner: {{ username }}
    group: {{ username }}
    content: |
      sudo username: {{ username }}
      sudo password: {{ password }}
```
write creds only to sudo user home. set owner and group to avoid the gotcha of root owning sudo creds.